subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_df.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <errno.h>
+#include <devicetree.h>
+#include <sys/util_macro.h>
+#include <nrfx/hal/nrf_radio.h>
+
+#include "radio_df.h"
+
+/* @brief Minimum antennae number required if antenna switching is enabled */
+#define DF_ANT_NUM_MIN 2
+/* @brief Value to set antenna GPIO pin as not connected */
+#define DF_GPIO_PIN_NOT_SET 0xFF
+/* @brief Number of PSEL_DFEGPIO registers in Radio peripheral */
+#define DF_PSEL_GPIO_NUM 8
+
+/* @brief Direction Finding antenna matrix configuration */
+struct df_ant_cfg {
+	uint8_t ant_num;
+	/* Maximum supported antenna switch pattern length */
+	uint8_t max_ant_sw_pattern_num;
```

Comment:
This field does not seem to be used in any way.

Reply:
I will delete it.

Reply:
corrected